.env.example

```diff
@@ -0,0 +1,7 @@
+REACT_APP_CANDY_MACHINE_CONFIG=__PLACEHOLDER__
+REACT_APP_CANDY_MACHINE_ID=__PLACEHOLDER__
+REACT_APP_TREASURY_ADDRESS=__PLACEHOLDER__
+REACT_APP_CANDY_START_DATE=__PLACEHOLDER__
+
+REACT_APP_SOLANA_NETWORK=devnet
+REACT_APP_SOLANA_RPC_HOST=https://explorer-api.devnet.solana.com
```

Comment:
thanks for adding this it was badly needed

Reply:
thanks for adding